src/data/pi.lean

```diff
@@ -40,18 +43,26 @@ instance has_mul [∀ i, has_mul $ f i] :
 
 @[to_additive] lemma mul_def [Π i, has_mul $ f i] : x * y = λ i, x i * y i := rfl
 
+@[to_additive] lemma mul_comp [has_mul γ] (h : α → β) (f g : β → γ) :
+  (f * g) ∘ h = f ∘ h * g ∘ h := rfl
```

Comment:
Can you use `x y z` in stead of `f g h` here, to match `mul_apply`, and not clash with the `f : I → Type`? Same throughout